app/src/services/conditional_token.ts

```diff
@@ -124,7 +131,7 @@ class ConditionalTokenService {
       collateralToken,
       ethers.constants.HashZero,
       conditionId,
-      [1, 2],
+      [1, 2], // TODO: analize if we need to refactor this
```

Comment:
I think this needs to be replaced with a `indexSets` variable, that is computed like this:

```typescript
const range = length => [...Array(length)].map((x, i) => i)
const indexSets = range(outcomesCount).map(x => 1 << x)
```

I think this comes from the fact that for N outcomes, you can describe a position with a bit array of length N, where something like `1010` means the second and fourth outcomes. But since we don't have conditional positions yet, all our index sets have only one 1. For two outcomes, the valid index sets are `01` and `10` (the `[1, 2]`) array that is being used right now). For N outcomes, you can use the bit shift operator to generate all of them.